cmake/dependencies/CDashSubprojectDependencies.xml

```diff
@@ -748,23 +748,6 @@
       <Email address="ctrilinos-regression@software.sandia.gov"/>
     </EmailAddresses>
   </SubProject>
-  <SubProject name="ForTrilinos">
-    <Dependency name="Teuchos" type="LR"/>
-    <Dependency name="Epetra" type="LR"/>
-    <Dependency name="Triutils" type="ILO"/>
-    <Dependency name="Tpetra" type="ILO"/>
-    <Dependency name="EpetraExt" type="ILO"/>
-    <Dependency name="Xpetra" type="ILO"/>
-    <Dependency name="Pliris" type="LO"/>
-    <Dependency name="AztecOO" type="LO"/>
-    <Dependency name="Galeri" type="LO"/>
-    <Dependency name="Amesos" type="LO"/>
-    <Dependency name="Ifpack" type="LO"/>
```

Comment:
Hum, this file should not even be under version control.   It gets generated automatically by TriBITS software on the fly. We should just remove it.